system-int-test/test/cmr/system_int_test/ingest/subscription_ingest_test.clj

```diff
@@ -43,6 +44,58 @@
                            (map #(select-keys % [:concept-id :extra-fields :metadata])))]
     (#'jobsub/process-subscriptions (system/context) subscriptions)))
 
+(deftest subscription-count-exceeds-limit-test
+  (side/eval-form `(jobsub/set-subscriptions-limit! 1))
+  (let [coll1 (data-core/ingest-umm-spec-collection
+               "PROV1"
+               (data-umm-c/collection
+                {:ShortName "coll-exceed-limit-1"
+                 :EntryTitle "entry-title-exceed-limit-1"})
+               {:token "mock-echo-system-token"})
+        coll2 (data-core/ingest-umm-spec-collection
+               "PROV1"
+               (data-umm-c/collection
+                {:ShortName "coll-exceed-limit-2"
+                 :EntryTitle "entry-title-exceed-limit-2"})
+               {:token "mock-echo-system-token"})
+        coll3 (data-core/ingest-umm-spec-collection
+               "PROV1"
+               (data-umm-c/collection
+                {:ShortName "coll-exceed-limit-3"
+                 :EntryTitle "entry-title-exceed-limit-3"})
+               {:token "mock-echo-system-token"})]
```

Comment:
I'm pretty sure it's possible to have multiple subscriptions on the same collection (as long as they have different queries).  I think it could be cleaner to drop these 2 collections and then just specify queries when calling `make-subscription-concept` (if you search in the same file for ":Query" you can see an example).  It's not actually important what the query is for the tests as long as they are valid queries.

Reply:
Thanks Brian, I removed coll2 and coll3 and just added all the test subscriptions to coll1 with their own queries.